realm/src/main/java/io/realm/internal/RealmCore.java

```diff
@@ -101,7 +101,7 @@ public static void gcGuaranteed(){
         Object obj = new Object();
         WeakReference<Object> ref = new WeakReference<Object>(obj);
         obj = null;
-        while(ref.get()!=null)
+        while (ref.get()!=null)
```

Comment:
You don't like spaces around the operators?
```while (ref.get() != null)```
vs
```while (ref.get()!=null)```

The first is a lot more readable imo